.gitattributes

```diff
@@ -17,3 +17,25 @@
 # Use `binary` to make sure certain files are never auto-detected as text.
 #
 #*.png binary
+
+# Directory symlinks
+include/freetype symlink=dir
```

Comment:
Shouldn't this be `include/freetype/internal`?

Reply:
Yes I think I made a mistake when I made the commit - I got the list from a tool I wrote that list the symlinks to directories, I probably mis-copied the output.
I'll fix it. Did you try by adding `internal` see if it worked on your side?

Reply:
freetype works with these changes for me. Also have a look at the new SFML master, Windows build (including freetype) worked fully on my setup. Waiting for the CI build there.

Reply:
I just tried locally and I see sfml master takes this branch and indeed it seems to work better - except like for this branch I hit the auto-conf build-module issue missing. I already reported this issue to the team so I think it will be fixed in stage soon.